client/src/pages/SignUp/SignUp.js

```diff
@@ -158,7 +158,7 @@ const SignUp = () => {
                 />
               </Grid>
               <Grid item xs={12}>
-                <Tooltip title="Password must be 16 characters long" placement="left">
+                <Tooltip title="Passphrase must be 16 characters long" placement="left">
```

Comment:
we probably want to change the name/label for the input fields as well if we make this change?

Reply:
Yeah good call